visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.styles.ts

```diff
@@ -1,214 +1,254 @@
-  addedFilter:IStyle;
-  addedFilterDiv:IStyle;
-  filterIcon:IStyle;
-  cohortName:IStyle;
-  saveCohort:IStyle;
-  saveAndCancelDiv:IStyle;
-  saveFilterButton:IStyle;
-  cancelFilterButton:IStyle;
+    wrapper: IStyle;
+    leftHalf: IStyle;
+    rightHalf: IStyle;
+    detailedList: IStyle;
+    filterHeader: IStyle;
+    addFilterButton: IStyle;
+    featureTextDiv: IStyle;
+    featureComboBox: IStyle;
+    operationComboBox: IStyle;
+    valueSpinButton: IStyle;
+    valueSpinButtonDiv: IStyle;
+    minSpinBox: IStyle;
+    maxSpinBox: IStyle;
+    featureText: IStyle;
+    treatCategorical: IStyle;
+    defaultText: IStyle;
+    existingFilter: IStyle;
+    filterLabel: IStyle;
+    defaultFilterList: IStyle;
+    container: IStyle;
+    addedFilter: IStyle;
+    addedFilterDiv: IStyle;
+    filterIcon: IStyle;
+    cohortName: IStyle;
+    saveCohort: IStyle;
+    saveAndCancelDiv: IStyle;
+    saveFilterButton: IStyle;
+    cancelFilterButton: IStyle;
+    closeIcon: IStyle;
+    cohortEditor: IStyle;
 }
 
 export const cohortEditorStyles: () => IProcessedStyleSet<ICohortEditorStyles> = () => {
     const theme = getTheme();
     return mergeStyleSets<ICohortEditorStyles>({
+        cohortEditor: {
+            position: 'absolute',
+            overflowY: 'visible',
+            width: '560px',
+            height: '624px',
+            left: '500px',
+            top: '100px',
+            //TODO: fix rgba here
+            boxShadow: '0px 0.6px 1.8px rgba(0, 0, 0, 0.108), 0px 3.2px 7.2px rgba(0, 0, 0, 0.132)',
+            borderRadius: '2px'
+        },
         wrapper: {
             height: "344px",
             display: "flex",
-            marginTop:"7px"
+            marginTop: "7px"
         },
         leftHalf: {
             height: "344px",
-            marginLeft:"40px"
+            marginLeft: "40px",
+            width: "213px"
         },
         rightHalf: {
             display: "flex",
             width: "255px",
-            height:"344px,",
+            height: "344px,",
             flexDirection: "column",
-            background:theme.palette.neutralLight,
-            marginRight:"27px",
-            marginLeft:"25px",
-            marginTop:"0px",
+            background: theme.palette.neutralLight,
+            marginRight: "27px",
+            marginLeft: "25px",
+            marginTop: "0px",
             borderRadius: "5px"
         },
         detailedList: {
-            marginTop:"28px",
-            height:"160px",
-            width:"213px"
+            marginTop: "28px",
+            height: "160px",
+            width: "197px",
+            overflowX: "visible"
         },
-        filterHeader:{
-                fontWeight: FontWeights.semibold,
-                fontSize: FontSizes.medium,
-                color:theme.palette.black
+        filterHeader: {
+            fontWeight: FontWeights.semibold,
+            fontSize: FontSizes.medium,
+            color: theme.palette.black
```

Comment:
for text we may want to use semanticColor.<some text label> to have it work in dark and high contrast. This will present itself when we start dark theme work, so not to worry about now